### DISCUSSION TITLE
Join forces with another project

### DISCUSSION THREAD

**Discussion by EliranTurgeman** [jheredianet/Teslamate-CustomGrafanaDashboards#22] [General]
While searching for additional dashboards for TeslaMate I saw your project and another good one:

https://github.com/CarlosCuezva/dashboards-Grafana-Teslamate

I suggest to join both project into one 💪🏼
🙂

**Comment by jheredianet** [comment]
Hi @EliranTurgeman!
I'm glad you found this. 
About joining, maybe soon, since I'm already in other personal projects and running out of time it's not possible now. 
In the meantime enjoy these ones.

Regards,
